cmd/ecs-service-deployer/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/trussworks/truss-aws-tools/internal/aws/session"
+	"github.com/trussworks/truss-aws-tools/pkg/ecsdeploy"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	flag "github.com/jessevdk/go-flags"
+	"go.uber.org/zap"
+)
+
+// Options are the command line options
+type Options struct {
+	ECSCluster string `long:"ecs-cluster-identifier" description:"The ECS cluster identifier." required:"true" env:"ECS_CLUSTER"`
+	ECSService string `long:"ecs-service-identifier" description:"The ECS service identifier." required:"true" env:"ECS_SERVICE"`
+	DryRun     bool   `long:"dry-run" description:"Don't make any changes and log what would have happened." env:"DRY_RUN"`
+	Lambda     bool   `long:"lambda" description:"Run as an AWS lambda function." required:"false" env:"LAMBDA"`
+	Profile    string `long:"profile" description:"The AWS profile to use." required:"false" env:"PROFILE"`
+	Region     string `long:"region" description:"The AWS region to use." required:"false" env:"REGION"`
+
+	Args struct {
```

Comment:
I saw in your PR walkthrough that you were passing in a path to a local JSON file (?), but I'm a little confused where that JSON would be coming from originally, or if those args could be passed in as separate option values?

Reply:
Ah, so the JSON could theoretically come from a call to the api to get a task definition and submitted as a complete set of container definitions someday.

I suppose I leave it as a JSON because there may be multiple containers with images to "reconfigure" in a task definition. I think it would be harder to understand what was being passed if they were passed as args.

For example:
{
    "containers":
    {
        "atlantis": {
            "image": "runatlantis/atlantis:latest"
        },
        "nginx":{
            "image": "someimagepathhere"
        }
    }
}
vs command --ECSCluster atlantis-prod --ECSService atlantis --containernames atlantis,nginx --containerimages runatlantis/atlantis:latest,  someimagepathhere

Reply:
ah I see, that makes a lot of sense, thank you for explaining :)